.gitignore

```diff
@@ -16,6 +16,7 @@ DerivedData/
 *.perspectivev3
 !default.perspectivev3
 xcuserdata/
+xcshareddata/
```

Comment:
Ohh, This also shouldn't be added to this place! You can see [#2585](https://github.com/SDWebImage/SDWebImage/pull/2585) firstly.